### ISSUE TITLE
Existing download codepath acceleration

### ISSUE THREAD

**Issue by jtolio** [storj/uplink#157]
Understanding we plan to do a larger download codepath refactor, there is a specific performance problem with the multicore servers in Durham right now, and so we want a small, targeted intervention to accelerate download speeds that reduce thrashing due to contention.

**Comment by iglesiasbrandon** [comment]
@jtolio is this ticket considered completed? Were we able to resolve the performance issues observed on the Durham setup? if so what was the solution?

**Comment by jtolio** [comment]
yes, and yes! we rewrote the part that manages concurrent piece downloads to be much faster.

**Comment by onionjake** [comment]
Do we have another ticket to enable this on edge?

**Comment by jtolio** [comment]
it's enabled!
